_posts/current/blog/2020-11-06-accelerating-libcamera-qcam-format-conversion-using-opengl-shaders.md

```diff
@@ -346,4 +345,4 @@ After moving the format conversion to the GPU, the qcam frame rate improved a lo
 
 ## About the Author
 
-Show is an engineer within the Multimedia Working Group. For further information on this group, click [here](https://www.linaro.org/engineering/multimedia).
\ No newline at end of file
+Show is an engineer within the Multimedia Working Group. For further information on this group, click [here](https://www.linaro.org/client-devices/).
```

Comment:
Please remove `https://www.linaro.org`, leaving it as just `/client-devices`.

That will make things work better in the link checker as it then looks within the built site rather than trying to reference something that is "currently" live on the web.